airflow/providers/airbyte/hooks/airbyte.py

```diff
@@ -50,6 +61,53 @@ def __init__(self, airbyte_conn_id: str = "airbyte_default", api_version: str =
         super().__init__(http_conn_id=airbyte_conn_id)
         self.api_version: str = api_version
 
+    async def get_headers_tenants_from_connection(self) -> tuple[dict[str, Any], str]:
+        """Get Headers, tenants from the connection details."""
+        connection: Connection = await sync_to_async(self.get_connection)(self.http_conn_id)
+        base_url = connection.host
+
+        credentials = f"{connection.login}:{connection.password}"
+        credentials_base64 = base64.b64encode(credentials.encode("utf-8")).decode("utf-8")
+
+        authorized_headers = {
+            "accept": "application/json",
+            "content-type": "application/json",
+            "authorization": f"Basic {credentials_base64}",
+        }
+
+        return authorized_headers, base_url
+
+    async def get_job_details(self, job_id: int) -> Any:
+        """
+        Uses Http async call to retrieve metadata for a specific job of an Airbyte Sync.
+
+        :param job_id: The ID of an Airbyte Sync Job.
+        """
+        headers, base_url = await self.get_headers_tenants_from_connection()
+        url = f"{base_url}/api/{self.api_version}/jobs/get"
+        self.log.info("URL for api request: %s" % url)
+        async with aiohttp.ClientSession(headers=headers) as session:
+            async with session.post(url=url, data=json.dumps({"id": job_id})) as response:
+                try:
+                    response.raise_for_status()
+                    return await response.json()
+                except ClientResponseError as e:
+                    raise AirflowException(str(e.status) + ":" + e.message + "-" + str(e.request_info))
+
+    async def get_job_status(self, job_id: int) -> str:
+        """
+        Retrieves the status for a specific job of an Airbyte Sync.
+
+        :param job_id: The ID of an Airbyte Sync Job.
+        """
+        try:
+            self.log.info("Getting the status of job run %s.", str(job_id))
```

Comment:
`%s` converts a specified value to a string using the `str()` function itself, see: https://docs.python.org/3/library/stdtypes.html#printf-style-string-formatting

```suggestion
            self.log.info("Getting the status of job run %s.", job_id)
```